tests/ssl-opt.sh

```diff
@@ -883,9 +901,13 @@ print_name() {
 
     LINE="$LINE$1"
     printf "%s " "$LINE"
```

Comment:
There's an extra space at the end in `LIST_TESTS` mode. It's no big deal, but it would be simpler to just do
```
if [ "$LISTS_TESTS" -eq 0 ]; then
    printf '%s\n' "$LINE"
    return
fi
```

Reply:
Done

Reply:
Done